src/schedule.md

```diff
@@ -1,55 +1,55 @@
 # Schedule
 
-## Winter/Spring 2023
+## Fall 2023
 
 <div class="timeline" markdown="1">
 
-- ### [January 19th, 2023, Thursday](./archive/2023/2023-01-19.md)
+- ### [September 21st, 2023, Thursday](./archive/2023/2023-09-21.md)
 
-    :material-clock: 6:00pm - 9:00pm PT
+    :material-clock: 11:00am - 1:30pm PT
 
-    :material-map-marker:  St. Paul's Hospital: [Cullen Family Lecture Theatre (Room 1477)](https://docs.google.com/document/d/1xHHd14LcrDIZLG7RGBGneLgf12H-FJwpyH7rotQCo9k/edit?usp=sharing)
+    :material-map-marker: SFU, Big Data Hub, room ASB 10900
 
-    Featured Speaker: **Dr. Heng Li**
+    :material-video: UBC, Michael Smith Laboratories, MSL 102 (Live Stream Location)
 
-    Introductory Speaker: **Dr. Melissa Chen (Cara Haney Lab)**
+    Featured Speaker: **Dr. Keegan Korthauer**
 
-- ### [February 16th, 2023, Thursday](./archive/2023/2023-02-16.md)
+    Introductory Speaker: TBA
```

Comment:
Other sections say "Trainee Speaker:" I think that should be the case here. The only reason it wasn't for the previous schedule is the intro speaker ended up being a PhD and it didn't seem appropriate to label her a "Trainee".